### ISSUE TITLE
Response body tab isn't updated

### ISSUE THREAD

**Issue by paulhitz** [paulhitz/yet-another-rest-client#17]
If the request response is numeric, then the response body tab isn't updated. The previous response remains.

**Comment by paulhitz** [comment]
This is the culprit: 	helper.isValidResponse

The response doesn't have to be an object.

**Comment by paulhitz** [comment]
The syntax highlighting library seemingly can't handle numeric inputs.

```
TypeError: value.replace is not a function
    at escape (highlight.pack.js:25)
    at Object.highlightAuto (highlight.pack.js:502)
    at d.highlight (angular-highlightjs.min.js:6)
    at Object.fn (angular-highlightjs.min.js:6)
    at m.$digest (angular.js:15715)
    at m.$apply (angular.js:15986)
    at g (angular.js:10511)
    at L (angular.js:10683)
    at XMLHttpRequest.A.onload (angular.js:10624)
```

A potential solution would be to check if the response body is numeric (angular.isNumber). If so, change it to a String.

**Comment by paulhitz** [comment]
Fixed in 0.0.3.

Numeric responses are now converted to Strings.
